ariadne/cardano/src/Ariadne/Wallet/Backend.hs

```diff
@@ -110,8 +110,11 @@ createWalletBackend walletConfig sendWalletEvent getPass voidPass = do
                     sendTx pwl this cf walletSelRef putCommandOutput getPassPhrase voidWrongPass waitUiConfirm
                 , walletBalance = getBalance pwl walletSelRef
                 }
-            initWalletAction =
-                refreshState pwl walletSelRef sendWalletEvent
+            initWalletAction = refreshState pwl walletSelRef sendWalletEvent
+
+            walletAction = do
```

Comment:
@gromakovsky said that our main template should not know about what exactly this function doing. At least because we can add some more functions to the walletAction in the future.

Reply:
Well, we can call it `postStartupAction` at least, I suppose. Again, `walletAction` is non-descriptive.

Reply:
Okay, let's call it `postStartupAction` as the middle ground. Or `postInitAction`.